app/components/provider_interface/activity_log_event_component.rb

```diff
@@ -96,6 +98,8 @@ def link
         feedback_link
       elsif event.audit.auditable.is_a?(Interview)
         interview_link(event.audit)
+      elsif changes['course_changed_at'].present?
```

Comment:
I don't think we need this as we already default to an application link which is similar